try-cb-dotnet/Services/HotelService.cs

```diff
@@ -50,9 +51,9 @@ public async Task<IEnumerable<dynamic>> FindHotel(string description, string loc
                     new PhraseQuery(location).Field("country")
                 ));
             }
-            var Q = new SearchQuery { Query = query };
+            // var Q = new SearchQuery { Query = query };
 
-            var queryString = (Q).ToJson();
+            // var queryString = (Q).ToJson();
```

Comment:
Same here

Reply:
I think the API's changed, I commented as wasn't quite sure if this would need to be reintroduced, but I'll tidy up when I get to the next step (e.g. actually running the code and checking it works :-) )

Reply:
For the record, this code *doesn't work*, but it looks like it didn't work previously (as per commit history), so this will be fun to debug :D